index.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+$db = new PDO('mysql:host=192.168.20.20; dbname=collectorsapp', 'root', '');
+$db->setAttribute(
+    PDO::ATTR_DEFAULT_FETCH_MODE,
+    PDO::FETCH_ASSOC);
+
+$sql = 'SELECT * FROM `grocery_item`';
+$sql = $db->prepare('SELECT * FROM `grocery_item`;');
+$sql->execute();
+$groceryItem = $sql->fetchAll();
+
+//var_dump($groceryItem);
+
+foreach($groceryItem as $item) {
```

Comment:
Similarly, I would suggest putting your display into a function as well, so it can be unit tested.

Reply:
ok changed to funtion